src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+$main: #060b35;
+$secondary: #616070;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  flex-direction: column;
+  width: 200px;
+  border: solid 1px #f3f3f3;
+  display: flex;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+  padding: 32px 16px 16px;
+}
```

Comment:
there should be 1 file for every BEM block - according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) point 11 - please create 1 file for stars and 1 for card, then import those in index.scss - you can also reset styles there